server/src/routes/api/v1/reviewsRouter.js

```diff
@@ -0,0 +1,20 @@
+import express from "express"
+import Review from "../../../models/Review.js"
+
+const reviewsRouter = new express.Router()
+
+reviewsRouter.delete("/:id", async (req, res) => {
+  try {
+    const review = await Review.query().findById(req.params.id)
+    if(req.user.id === review.userId){
+      await Review.query().deleteById(req.params.id)
+      res.status(200).json({ message: "This review was successfully deleted" })
+    } else{
+      res.status(401).json({ message: "You did not make this review, cannot delete" })
```

Comment:
the last bit is poor grammar. Update for better grammar! 

https://tenor.com/bkN0E.gif